lib/compilers/ldc.js

```diff
@@ -42,6 +42,26 @@ export class LDCCompiler extends BaseCompiler {
         this.compiler.supportsIntel = true;
         this.compiler.supportsIrView = true;
         this.compiler.irArg = ['-output-ll'];
+        
+        this.compiler.supportsLLVMOptPipelineView = true;
+        this.compiler.llvmOptArg = ['--print-after-all', '--print-before-all'];
+        this.compiler.llvmOptModuleScopeArg = ['--print-module-scope'];
+        this.compiler.llvmOptNoDiscardValueNamesArg = []; // LDC does not have a flag for this yet.
+        
+        const asanSymbolizerPath = path.dirname(this.compiler.exe) + '/llvm-symbolizer';
```

Comment:
can you use path.join() here instead of concatting

Reply:
i copied this from clang.js.  path.join(this.compiler.exe, '/llvm......') ?

Reply:
you're right, it's only in some really old version from 2017 but not the more recent ones

Reply:
Is it possible to refer to a CE-wide llvm-symbolizer that any compiler/language can use?

Reply:
Definitely, will make a change tonight to use one of the clang installations when im behind laptop